pages/configuring_webif.rst

```diff
@@ -308,106 +308,127 @@ If you want to run a load balancer/reverse proxy in front of Graylog, you need t
+            proxy_set_header    X-Forwarded-Server $host;
+            proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header    X-Graylog-Server-URL http://graylog.example.org/api;
+            proxy_pass          http://127.0.0.1:9000;
+        }
+    }
 
-  server
-  {
-    listen      443 ssl spdy;
-    server_name graylog.example.org;
+**REST API and web interface on separate ports (using HTTP)**::
 
-    location /
+    server
     {
-        proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
-        proxy_set_header    Host $http_host;
-        proxy_set_header    X-Graylog-Server-URL https://graylog.example.org/api;
-        proxy_pass          http://graylog.internal.example.org:9000;
+        listen      80 default_server;
+        listen      [::]:80 default_server ipv6only=on;
+        server_name graylog.example.org;
+
+        location /
+        {
+            proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header    X-Graylog-Server-URL http://graylog.example.org:12900;
+            proxy_set_header    Host $http_host;
+            proxy_pass          http://127.0.0.1:9000;
+        }
     }
 
-    location /api/
+    server
     {
-        proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
-        proxy_set_header    Host $http_host;
-        proxy_pass          http://graylog.internal.example.org:12900/;
+        listen      12900;
+        server_name graylog.example.org;
+
+        location /
+        {
+            proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header    Host $http_host;
+            proxy_pass          http://127.0.0.1:12900/;
+        }
     }
-  }
 
 
-**REST API and web interface on separate ports (using HTTPS/SSL)**::
+NGINX can be used for SSL Termination, you would only need to modify the ``server listen`` directive and add all Information about your certificate.
 
-  server
-  {
-    listen      443 ssl spdy;
-    server_name graylog.example.org;
+If you are running multiple Graylog Server you might want to use HTTPS/SSL to connect to the Graylog Servers (on how to Setup read :ref:`ssl_setup`) and use HTTPS/SSL on NGINX. The configuration for TLS certificates, keys and ciphers is omitted from the sample config for brevity's sake.
 
-    location /
-    {
-        proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
-        proxy_set_header    X-Graylog-Server-URL https://graylog.example.org:12900;
-        proxy_set_header    Host $http_host;
-        proxy_pass          http://graylog.internal.example.org:9000;
-    }
-  }
+**REST API and Web Interface on one port (using HTTPS/SSL)**::
 
-  server
-  {
-    listen      12900 ssl spdy;
-    server_name graylog.example.org;
+    server
+    {
+        listen      443 ssl spdy;
+        server_name graylog.example.org;
+        # <- your SSL Settings here!
 
     location /
```

Comment:
Minor nitpick: the indentation of the `location` directive is one level too high (should be the same as the curly braces).